src/cswinrt/strings/additions/Windows.Storage.Streams/IBufferByteAccess.cs

```diff
@@ -18,6 +18,24 @@ namespace ABI.Windows.Storage.Streams
     using global::System.Runtime.InteropServices;
     using global::System.ComponentModel;
 
+    internal static class IBufferByteAccessMethods
```

Comment:
Where are these new memberes being used?

Reply:
They are no longer used, removed. Just need to keep the IID.